### ISSUE TITLE
MODX upgrade should respect existing dashboard widgets

### ISSUE THREAD

**Issue by ant-tar** [modxcms/revolution#13981]
### Summary
Dashboard widget list was changed due to MODX upgrade

### Step to reproduce
Set some other widget list (inside default Dashboard) so that if will be different from default one. Then upgrade MODX from 2.6.0-pl up to 2.6.4-pl and check your Default Dashboard settings again.

### Observed behavior
I had only BigBrother widget for Dashboard, please see how it was: 

![dashboard _ before google chrome 2018-07-09 01 12 22](https://user-images.githubusercontent.com/2815640/42442312-ff037d22-8372-11e8-91d3-df803a719ded.png)

![dashboards _ before_db - google chrome 2018-07-09 01 13 45](https://user-images.githubusercontent.com/2815640/42442314-ff24aeb6-8372-11e8-95c3-3c5800f19aff.png)

And here is what I've got after upgrade:

![dashboard _ db_after2 - google chrome 2018-07-09 01 17 16](https://user-images.githubusercontent.com/2815640/42442310-feab08fe-8372-11e8-9b0e-d41f1576dea0.png)

![dashboards _db_after- google chrome 2018-07-09 01 15 59](https://user-images.githubusercontent.com/2815640/42442311-fed53b38-8372-11e8-8469-dd25c4b70581.png)

### Expected behavior
I believe dashboard settings should not be supplemented or overwritten, everything there should remain the same as it was set before.

### Environment
MODX 2.6.0-pl on MODXCloud

**Comment by jonleverrier** [comment]
If wonder if the solution here is to not use the "Default" dashboard if you don't want the settings to 
be overwritten during upgrades?

Create a new custom default dashboard that will be safe from upgrades.

**Comment by ant-tar** [comment]
That's right @jonleverrier , will do so! But I've never seen any information (please let me know if exists) that  dashboard settings can be overwritten ... It seems strange to me, but if this is feature and not bug - nps,  just observation 👍

**Comment by AlexUnicom** [comment]
Please, tell me how could I update my MODX Revolution 2.2.14-pl (traditional)? There are not a point of modx upgrade or something like it in the package grid. My Modx was hacked with virus soft. I restored it. But it is very needed to upgrade modx and some plugins. For some reason when I updated all plugins my admin console is collapse or appear bugs on my site.

Thanks in advance

**Comment by Mark-H** [comment]
@AlexUnicom https://docs.modx.com/revolution/2.x/administering-your-site/upgrading-modx

The UpgradeMODX extra, available from the package manager, can help make it easier.

**Comment by AlexUnicom** [comment]
Dear Mark, Thank you for help

**Comment by AlexUnicom** [comment]
Hello guys! Can somebody help me to resolve it? If I upgrade my MODX Revolution 2.2.14-pl (traditional) to   2.6.5-pl version the catalogs on site do not work. And if update FormIt extra then an error appears on screen only.

**Comment by Mark-H** [comment]
@AlexUnicom We can't help you with that in the issue tracker. Please post on the forum if you need help with that.

**Comment by Ibochkarev** [comment]
### Triage (current 3.x)

Mostly a packaging/upgrade policy issue. Comments suggested using a custom dashboard instead of editing Default.

3.x upgrade scripts touch dashboard *schema* (e.g. `3.0.0-dashboard-widgets.php`) more than blindly rewriting placements, but core transport still ships default widgets/dashboard. Re-running package install / upgrades can still reshape the stock Default dashboard depending on vehicle attributes.

Practical advice from the thread still holds: don’t customize the stock Default if you need it stable across upgrades. Keep open only if current 3.x upgrades still re-attach default widgets onto an already-customized Default dashboard; otherwise close as expected transport behavior + docs.
